server/form-pages/apply/move-on/placementDuration.test.ts

```diff
@@ -106,22 +106,44 @@ describe('PlacementDuration', () => {
       })
     })
 
-    it('returns an error if the different duration response is yes but both durations are not set', () => {
-      let page = new PlacementDuration(
-        { differentDuration: 'yes', durationWeeks: '1', reason: 'Some reason' },
+    it.each([
```

Comment:
I'm thinking this is overkill as the same tests are applied to `validWeeksAndDaysDuration` All that really needs doing here is to test the generated error message.  Mock validWeeksAndDaysDuration perhaps?

Reply:
Oh of course -- I think I added it here first then realised it could be shared elsewhere. I'll get rid, and instead spy and check it's called as suggested.

Reply:
I've updated the above now, and also updated the test for the other handler that uses the same validation -- that was also doing a whole lot of logic testing, which wasn't necessary.